js/core/css/respec2.css

```diff
@@ -39,6 +39,13 @@ a.bibref {
   text-decoration: none;
 }
 
+a.bibref:target {
```

Comment:
We may want the `:target` css to be applied to more places, not just `a.bibref`

Reply:
so i should just change a.bibref:target to target??

Reply:
@marcoscaceres WDYT?

Reply:
Also, we may want to consider Wikipedia like `:target` styling. It's simpler :slightly_smiling_face:

Reply:
hey just set the background-color to #eaf3ff :smile:

Reply:
Lets do that :smiley:

Reply:
:+1:  on all :target